include/qpx_mmc.hpp

```diff
@@ -142,9 +142,9 @@ typedef struct {
   uint8_t max_rate;
   uint8_t disc_size; // indicates 120/80mm disc
   uint8_t polarity;  // Push-Pull polarity flags per layer for BD (indicates HtL
-                     // or LtH)
-  uint8_t layers;    // Layers num (!CD)
-  uint8_t gbpl;      // Layers num (!CD)
+  // or LtH)
+  uint8_t layers; // Layers num (!CD)
+  uint8_t gbpl;   // Layers num (!CD)
```

Comment:
Codacy found an issue: [struct member 'media_info::gbpl' is never used.](https://app.codacy.com/manual/jeremiahwilson777/qpxtool/pullRequest?prid=6191247)